SU-SWS/stanford_events_importer
# Change past events filter from <= to <

#  READY FOR REVIEW

# Summary
- This PR is in response to [issue #80](https://github.com/SU-SWS/stanford_events_importer/issues/80).  I adjusted the operator on the filter for the Past Event page from <= to <. Thereby stopping events that happen now or today from appearing on the Past Events page.

# Needed By (Date)
- When possible

# Criticality
- Fixes broken stuff


# Steps to Test
- Switch to this branch
- Flush cache
- Create a test event that happens today
- Navigate to `events/upcoming-events` 
- Verify you see your test event
- Navigate to `events/past-events`
- Verify you _don't_ see your test event

# Affects 
- All products

# Associated Issues and/or People
## Related Issue
https://github.com/SU-SWS/stanford_events_importer/issues/80

## Related PRs

## More Information

## Folks to notify
@eolienne 


# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
